olive/model/__init__.py

```diff
@@ -551,7 +551,14 @@ def load_mlflow_model(self):
 
         with open(os.path.join(self.model_path, "MLmodel"), "r") as fp:
             mlflow_data = yaml.safe_load(fp)
-            hf_pretrained_class = mlflow_data["flavors"]["hftransformers"]["hf_pretrained_class"]
+            # default flavor is "hftransformersv2" name from azureml.evaluate.mlflow>=0.0.8
+            # "hftransformers" is the name from azureml.evaluate.mlflow<0.0.8
+            # TODO: let user specify flavor name if needed
+            flavors_names = ["hftransformersv2", "hftransformers"]
+            for flavors_name in flavors_names:
+                if flavors_name in mlflow_data["flavors"]:
+                    hf_pretrained_class = mlflow_data["flavors"][flavors_name]["hf_pretrained_class"]
```

Comment:
if the first key doesn't exist, the KeyError will be raise.

Reply:
flavors always exits.

Reply:
hftransformersv2

Reply:
Fixed.